### PR TITLE
JK 11-end

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
So some things look a little funny but I wasn't sure what to do. I asked one of the questions in Slack because maybe you could find a better way to format it. The beginning of the stage entertainment chapter I wasn't sure about, and the end of the testimonies chapter I wasn't sure about either. :)

File: journal-letters/modernized/15-considerations-on-time.adoc [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+____
+For you know perfectly, that the day of the Lord so comes as a theif in the night.
+____
+
+[quote.scripture, , ]
+____
+Therefore let us watch, and be sober.
+____
+
+[quote.scripture, , 1 Thess. 5.]
```
Comment (jaredh159) [inline journal-letters/modernized/15-considerations-on-time.adoc]: yeah, i played around a bit with this on my computer, and i think it's both more correct and looks slightly better to make all 3 of these `quote.section-epigraph`

File: journal-letters/modernized/15-considerations-on-time.adoc [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+____
+Let us who are of the day, be sober, putting on the breast-plate of Faith and Love,
+and for a helmet, the Hope of Salvation.
+____
+
+[.offset]
```
Comment (jaredh159) [inline journal-letters/modernized/15-considerations-on-time.adoc]: And this is a pretty good (i think) use-case for `.section-date`

File: journal-letters/modernized/16-on-stage-entertainment.adoc [hunk 3]
```diff
@@ -1,12 +1,13 @@
 [short="Remarks on Attending Stage Entertainments"]
 == Remarks on the Prevailing Custom of Attending Stage Entertainments; also on the Present Taste for Reading Romances and Novels; and on some other Customs
 
-Submitted, with a heart overflowing with good-will,
-to the notice and consideration of the Professors of the Christian name,
-in the different religious Societies.
+[.blurb]
```
Comment (jaredh159) [inline journal-letters/modernized/16-on-stage-entertainment.adoc]: Ok, so this whole intro section was pretty difficult.  I definitely think that this "Submitted..." chunk qualifies as a "Heading Continuation Blurb" (which is in the styleguide, but rarely used -- and often I mark those up before they get to you.

Anyway, I pulled this down and tried a few variations and I think I'll have you while your making the other changes, just change this whole section to what I paste below:

```ASCIIDOC
== Remarks on the Prevailing Custom of Attending Stage Entertainments

[.heading-continuation-blurb]
Also on the Present Taste for Reading Romances and Novels; and on some other Customs

[.heading-continuation-blurb]
Submitted, with a heart overflowing with good-will,
to the notice and consideration of the Professors of the Christian name,
in the different religious Societies.

[.centered]
_Vitarc peccatum, est vitare occasioncs peccati._

[.section-author-context]
Reprinted from the Third Edition.

=== Remarks, etc.
```

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Thanks Maija, nice work on some difficult sections.  I left a few comments for a couple of changes. 👍

**Review by jaredh159 — APPROVED** [review]

**Comment by jaredh159** [comment]
thanks maija, merged 👍

**Pull request merged**